client/java/src/main/java/com/linecorp/centraldogma/client/AbstractWatcher.java

```diff
@@ -247,6 +296,7 @@ private void doWatch(int numAttemptsSoFar) {
                  logger.debug("watcher noticed updated file {}/{}{}: rev={}",
                               projectName, repositoryName, pathPattern, newLatest.revision());
                  notifyListeners(newLatest);
+                 latestReceivedTimeSeconds.set(Instant.now().getEpochSecond());
```

Comment:
nit: Should we use `volatile long` instead? For now, we don't get any advantage from `AtomicLong`.

Reply:
Ah, that's right.